config/notifies.go

```diff
@@ -15,62 +15,50 @@ package config
 
 import (
 	"fmt"
+	"strings"
 )
 
 var (
 	// DefaultHipchatConfig defines default values for Hipchat configurations.
 	DefaultHipchatConfig = HipchatConfig{
-		ColorFiring:   "purple",
-		ColorResolved: "green",
+		Color:         `{{ if eq .Status "firing" }}purple{{ else }}green{{ end }}`,
```

Comment:
This is the mix of template files and inlined templating I'm not yet sure about.
Then again, this is one of a bunch of configs taken from the old implementation that are not backed by an implementation.

Maybe it makes sense to remove these as well to start unbiased by the old structure when adding new integrations.